src/main/java/Duke.java

```diff
@@ -1,10 +1,100 @@
+    private static int numOfTasks = -1;
+    private static final int MAX_TASKS = 100;
+
+    private static Task[] tasks = new Task[MAX_TASKS];
+
+    private static void doneCommand(String input) {
+        int pos = Integer.parseInt(input.split(" ")[1]) - 1;
+        if (pos > numOfTasks) {
+            System.out.println(DOTTED_LINE);
+            System.out.println("No such task exists");
+            System.out.println(DOTTED_LINE);
+        } else {
+            tasks[pos].setDone();
+            System.out.println(DOTTED_LINE);
+            System.out.println("Nice I've marked this task as done:");
+            System.out.println(tasks[pos]);
+            System.out.println(DOTTED_LINE);
+        }
+    }
+
+    private static void listCommand() {
+        System.out.println(DOTTED_LINE);
+        for (int i = 0; i < numOfTasks; i++) {
+            System.out.println(i + 1 + "." + tasks[i]);
+        }
+        System.out.println(DOTTED_LINE);
+    }
+
+
+    private static void addCommand(String input) {
+        if (numOfTasks == -1) {
+            numOfTasks = 0;
+        }
+        System.out.println(DOTTED_LINE);
+        System.out.println("Got it. I've added this task: ");
+        String taskType = input.split(" ")[0];
+        int nameIndex = input.indexOf(' ');
+        int timeIndex;
+        switch (taskType) {
+        case "todo":
+            tasks[numOfTasks++] = new ToDo(input.substring(nameIndex + 1));
+            System.out.println(tasks[numOfTasks - 1]);
+            break;
+        case "deadline":
+            timeIndex = input.indexOf('/');
+            tasks[numOfTasks++] = new Deadline(input.substring(nameIndex + 1 , timeIndex), input.split("/")[1]);
+            System.out.println(tasks[numOfTasks - 1]);
+            break;
+        case "event":
+            timeIndex = input.indexOf('/');
+            tasks[numOfTasks++] = new Event(input.substring(nameIndex + 1 , timeIndex), input.split("/")[1]);
+            System.out.println(tasks[numOfTasks - 1]);
+            break;
+        default:
+            System.out.println("Invalid input type");
+        }
+        System.out.println("You now have " + numOfTasks + " items in the list.");
+        System.out.println(DOTTED_LINE);
+    }
+
+    private static void handleInput(String input) {
+        String command = input.split(" ")[0];
+        switch (command) {
+        case "done":
+            doneCommand(input);
+            break;
+
+        case "list":
+            listCommand();
+            break;
+
+        default: //Add is default function
+            addCommand(input);
+        }
+    }
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        String line;
```

Comment:
Maybe could give a better variable name to understand what it line means, eg userInput